server/subscribe_test.go

```diff
@@ -1381,7 +1381,6 @@ func TestGetChannelsSubscribed(t *testing.T) {
 			r := bytes.NewReader(data)
 			bb, err := ioutil.ReadAll(r)
 			require.Nil(t, err)
-
```

Comment:
Can you please remove this change, to keep the git history of the file more appropriate?